docs/tutorials/get-task-by-due-date.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+---
+
+# Tutorial: Get a task by due date
+
+In this tutorial, you learn the operations to call to
+get a task by the due date.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+## Before you start
+
+Make sure you've completed the [Before you start a tutorial](before-you-start-a-tutorial) topic on the development system you'll use for the tutorial.
+
+## Get a task by due date
+
+Finding a task by the due date requires that you retrieve (`GET`) the details of a [`task`](../api/task.md) using the `due_date` parameter.
+
+To get a task by due date:
+
+1. Make sure your local service is running, or start it by using this command, if it's not.
+
+    ```shell
+    cd <your-github-workspace>/to-do-service/api
+    json-server -w to-do-db-source.json
+    ```
+
+2. Open the Postman app on your desktop.
+3. In the Postman app, create a new request with these values:
+    * **METHOD**: GET
+    * **URL**: `{{base_url}}/tasks?due_date={due_date value}`
```

Comment:
The **\{\{base_url\}\}** text doesn't appear in the published page.

See `docs/tutorials/get-a-user-by-name.md` for an example of how to fix this, and then be sure to publish your docs from the feature branch to confirm it works.